content/en/docs/reference/config/networking/proxy-config/index.html

```diff
@@ -60,7 +60,7 @@
       app: ratings
   concurrency: 0
   image:
-    type: debug
+    imageType: debug
```

Comment:
Example: This comes from  https://github.com/istio/api/blob/release-1.14/networking/v1beta1/proxy_config.pb.html#L60-L61

Reply:
ok

Reply:
waiting for confirmation to update API or update documentation